styleguide/static/media/pastanaga.08f24832.svg

```diff
@@ -0,0 +1,83 @@
+

```

Comment:
Where this come from?

@ajayns @nileshgulia1 ?

Reply:
Same. removing...